src/local_scheduler/local_scheduler_client.cc

```diff
@@ -186,3 +187,37 @@ void local_scheduler_set_actor_frontier(LocalSchedulerConnection *conn,
   write_message(conn->conn, MessageType_SetActorFrontier, frontier.size(),
                 const_cast<uint8_t *>(frontier.data()));
 }
+
+std::pair<std::vector<ObjectID>, std::vector<ObjectID>> local_scheduler_wait(
+    LocalSchedulerConnection *conn,
+    const std::vector<ObjectID> &object_ids,
+    int num_returns,
+    int64_t timeout,
+    bool wait_local) {
```

Comment:
does `wait_local` work?

Reply:
No; I just added the piping for the argument. The back-end returns an unimplemented status if it's set to true. I have an idea of how to implement this whenever we'd like to add it.